src/main/resources/containers.json

```diff
@@ -1250,8 +1250,6 @@
     ]
   },
   {
-    "artifact_id": "glassfish",
-    "group_id": "org.glassfish.main.distributions",
     "name": "Arquillian Container GlassFish Managed",
```

Comment:
not only port, almost all properties which we are supporting for that container, user can configure it using forge. I think it's very good for user, as he don't want to search for any property if he want to override default property.